camus-shopify/script/upload_to_gcs.py

```diff
@@ -0,0 +1,199 @@
+#####
+# Upload directories imported by Camus to Google Cloud
+#####
+import argparse
+import os
+import subprocess
+import logging
+
+
+HDFS_PREFIX = "hdfs://hadoop-production"
+GCS_PREFIX = "gs://longboat"
```

Comment:
I've created `gs://raw-kafka` (and set up the right permissions)

Reply:
I'm happy to make the change, but it still distributes data between different buckets for downstream. Maybe we can discuss on Monday.